Procfile

```diff
@@ -0,0 +1,2 @@
+release: python manage.py migrate --no-input
```

Comment:
Shouldn't deploy.sh be executed here as well? We shouldn't be committing our React static files to our repo

Reply:
Added a section for generating the static files. testing it right now

Reply:
So heroku does not add Node/NPM to our container since it detects it as python. Manually adding the buildpack for node fails since the `package.json` is not in the root. We have two options.

1. say who cares and commit the static files 
1. move the react code to its own repo and deploy its own heroku app